### PR TITLE
Implement changes due to issue #1290

### PR DESCRIPTION
**Pull request by joergboe**

### CODE REVIEW

**Comment by ddebrunner** [comment]
In the future can the title/description of the PR describe what is being done so reviewers can determine if the changes match the intention.

**Comment by joergboe** [comment]
I changed the commit message

File: samples/python/com.ibm.streamsx.topology.pysamples/info.xml [hunk 1]
```diff
@@ -3,7 +3,16 @@
  xmlns:info="http://www.ibm.com/xmlns/prod/streams/spl/toolkitInfo">
 <info:identity>
   <info:name>com.ibm.streamsx.topology.pysamples</info:name>
-  <info:description>Sample toolkit for Python support</info:description>
+  <info:description>
+    Sample toolkit for Python support.
+    This sample demonstrates the usage of operators written in python in a Streams spl application.
```
Comment (ddebrunner) [inline samples/python/com.ibm.streamsx.topology.pysamples/info.xml]: nit: "Python" is the name of the language.

Comment (joergboe) [reply samples/python/com.ibm.streamsx.topology.pysamples/info.xml]: What does this mean in this concern?
Are there substantial objections?

**Comment by joergboe** [comment]
Is it possible to merge this PR now!

*    a build.xml in directory samples/python/com.ibm.streamsx.topology.pysamples
*    this build.xml calls the extract script and calls the sc to compile the samples.
*    the spl project settings are configured to use the 'external bulild' with this build.xml file
*    The info.xml of the sample explains the requirement to set the STREAMS_TOPOLOGY_TOOLKIT environment in oder to build the sample.

**Pull request merged**